platform/core/src/services/DisplaySetService/IDisplaySet.ts

```diff
@@ -1,9 +1,12 @@
-interface IDisplaySet {
+type IDisplaySet {
```

Comment:
If this is a type, we probably don't need to add the "I" prefix to "DisplaySet".